signer/src/config/default.toml

```diff
@@ -58,7 +58,7 @@ endpoints = [
 # Environment: SIGNER_BITCOIN__RPC_ENDPOINTS
 # Environment Example: http://user:pass@seed-1:4122,http://foo:bar@seed-2:4122
 rpc_endpoints = [
-    "http://devnet:devnet@localhost:18443",
+    "http://devnet:devnet@127.0.0.1:18443",
```

Comment:
I had to make this change otherwise on macOS `localhost` would be looking at IPv6 and failing.

We could decide to just stick to IPv4 on the default config.